packages/expo-in-app-purchases/ios/EXInAppPurchases/EXInAppPurchasesModule.m

```diff
@@ -151,6 +151,7 @@ - (void)handleQuery:(SKProductsResponse *)response {
   NSMutableArray *result = [NSMutableArray array];
   
   for (SKProduct *validProduct in response.products) {
+    if(!validProduct.localizedDescription) { continue; } // skip product with nil values - this can happen if it is in review "rejected" state
```

Comment:
```suggestion
    if (!validProduct.localizedDescription) { continue; } // skip product with nil values - this can happen if it is in review "rejected" state
```